### DISCUSSION TITLE
No access point after flash .bin file

### DISCUSSION THREAD

**Discussion by manhst** [revk/xESP32-Faikout#108] [General]
Hi my friend,
After i flash bin file in to the board. Try to restart the board but I can't see wifi access point. 
device is powered by GND and 3.3v from USB to serial pcb.

**Comment by revk** [comment]
Err, what does it say on the serial console?

**Reply by manhst** [reply]
Hi,
thank you so much for very quick response. I tried to use Hercules to capture serial response 
{0C}{A5}{4E}{8A}{F5}{94}{CD}{80}{41}{8D}{8C}{31}{84}{18}{9E}{DE}{F5}{0C}{A5}{E5}{85}{26}{0C}{CE}{8F}{28}{90}{BC}{21}{84}{D6}{9E}{43}{F5}{0C}{A5}{FF}{88}{ED}{D5}{94}{CD}{80}{61}{88}{31}{C6}{29}{A7}{1C}{FF}{0C}{A5}{E5}{85}{26}{0C}{CE}{84}{28}{90}{DE}{08}{29}{9C}{A5}{01}{A8}{0C}{A5}{E5}{85}{26}{0C}{CE}{84}{28}{90}{BC}{21}{84}{D6}{A5}{01}{A8}{0C}{A5}{FF}{88}{ED}{D5}{94}{01}{00}{88}{9E}{A4}{DC}{CE}{19}{42}{FF}{0C}{A5}{E5}{85}{26}{0C}{CE}{84}{28}{90}{BC}{6B}{21}{9C}{A5}{01}{A8}{0C}{A5}{E5}{85}{26}{0C}{CE}{84}{28}{90}{DE}{31}{86}{8D}{87}{9C}{F5}{0C}{A5}{FF}{88}{ED}{D5}{94}{CD}{C6}{29}{9E}{31}{C6}{21}{CE}{0E}

**Reply by revk** [reply]
The serial log should be text (debug/log), that is not text.

**Reply by manhst** [reply]
Thank you for your response. I think I knew what was problem. Because im using S3-Mini chip. And the exiting bin file is for S1.
here is error message
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0x7 (TG0WDT_SYS_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)
Saved PC:0x40043ac8
Invalid chip id. Expected 9 read 0. Bootloader for wrong chip?
ets_main.c 329 
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0x10 (RTCWDT_RTC_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)
Invalid chip id. Expected 9 read 0. Bootloader for wrong chip?
ets_main.c 329

**Reply by manhst** [reply]
Okay I manage to build bin file for S3, but got new error
ESP-ROM:esp32s3-20210327{0D}{0A}Build:Mar 27 2021{0D}{0A}rst:0x7 (TG0WDT_SYS_RST),boot:0x8 (SPI_FAST_FLASH_BOOT){0D}{0A}Saved PC:0x40050f3f{0D}{0A}SPIWP:0xee{0D}{0A}mode:DIO, clock div:2{0D}{0A}load:0x3c0e0020,len:0x4287c{0D}{0A}load:0x3fc99000,len:0x4ca0{0D}{0A}load:0x40374000,len:0x8acc{0D}{0A}load:0x42000020,len:0xd9cd4{0D}{0A}ESP-ROM:esp32s3-20210327{0D}{0A}Build:Mar 27 2021{0D}{0A}rst:0x10 (RTCWDT_RTC_RST),boot:0x8 (SPI_FAST_FLASH_BOOT){0D}{0A}SPIWP:0xee{0D}{0A}mode:DIO, clock div:2{0D}{0A}load:0x3c0e0020,len:0x4287c{0D}{0A}load:0x3fc99000,len:0x4ca0{0D}{0A}load:0x40374000,len:0x8acc{0D}{0A}load:0x42000020,len:0xd9cd4{0D}{0A}load:0x4037cacc,len:0xc45c{0D}{0A}load:0x00000000,len:0x9e70{0D}{0A}Guru Meditation Error: Core 0 panic'ed (StoreProhibited){0D}{0A}Core 0 register dump:{0D}{0A}PC      : 0x40045abc  PS      : 0x00060530  A0      : 0x80043ab9  A1      : 0x3fceb570{0D}{0A}A2      : 0x00000000  A3      : 0x3fceb670  A4      : 0x3fceb648  A5      : 0x0d2091c5{0D}{0A}A6      : 0x00000001  A7      : 0x3fceb55c  A8      : 0x00000000  A9      : 0x00000000{0D}{0A}A10     : 0x3c816160  A11     : 0x00000000  A12     : 0x00009e70  A13     : 0x00020000{0D}{0A}A14     : 0x00140000  A15     : 0x0013ffd0  SAR     : 0x00000005  EXCCAUSE: 0x0000001d{0D}{0A}EXCVADDR: 0x00000000  LBEG    : 0x40056f5c  LEND    : 0x40056f72  LCOUNT  : 0xffffffff{0D}{0A}{0D}{0A}Backtrace: 0x40045abc:0x3fceb570 0x40043ab6:0x3fceb6f0 0x40034c45:0x3fceb710

**Reply by manhst** [reply]
Hi @revk , I think the error is **Guru Meditation Error: Core 0 panic'ed (StoreProhibited)**
May I know what is problem?
Also can you help to build BIN file for S3-Mini :(

**Reply by revk** [reply]
That is a mess formatting, but no, I don't know what the problem is there.

**Reply by Rabbit1543** [reply]
Not sure if it will help but I had to disable psram via menuconfig to get my esp32s3 to boot.  FWIW this is the controller I plan in using. It comes with a 4 pin adapter that I have already soldered on and tested that it supplies power and boots l.Next step is to crimp a custom cable and install test the tx/rx. 
https://shop.m5stack.com/products/m5stamp-esp32s3-module
